README.md

```diff
@@ -36,7 +37,13 @@
 
 [](#__START_EMBEDDED_README__)
 
-
+## Docs
+To further document APIs that can be found in OverReact, the [doc](doc) directory was created. The documentation found in that directory includes:
+- [BuiltRedux to Redux](doc/built_redux_to_redux.md): A guide to transitioning to OverReact Redux from BuiltRedux.
```

Comment:
I think this information would easier to read if it were hierarchical:

> - OverReact Redux Documentation: The official documentation source for OverReact Redux, with an indepth description of connect and usage with UiComponent2.
>    
>    Migration guides from other state management libs:
>    - BuiltRedux to Redux: A guide to transitioning to OverReact Redux from BuiltRedux.
>    - Flux to Redux: A guide to how to transition from w_flux to OverReact Redux. This guide also introducers a new architecture, Influx, that can be used for incrementally refactoring.
>- Migration guides from older versions of over_react:
>    - Dart2 Migration: Documentation on the Dart 2 builder updates and how to transition componentry to Dart 2.
>    - UiComponent2 Transition: A document discussing the changes between UiComponent and UiComponent2, as well as how to migrate.

Reply:
Yep, that does make a lot more sense!